timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -445,6 +446,31 @@ private static void checkMethodArguments(JavaContext context, PsiMethodCallExpre
     }
   }
 
+  private static void checkExceptionLogging(JavaContext context, PsiMethodCallExpression call) {
+    PsiExpression[] arguments = call.getArgumentList().getExpressions();
+
+    if (arguments.length > 1) {
+      boolean isFirstParameterThrowable = isSubclassOf(context, arguments[0], Throwable.class);
+
+      if (isFirstParameterThrowable) {
+        PsiExpression secondArgument = arguments[1];
+        String message = findLiteralValue(secondArgument);
+
+        boolean callsGetMessage = false;
+
+        if (secondArgument instanceof PsiMethodCallExpression) {
+          PsiMethodCallExpression callExpression = (PsiMethodCallExpression) secondArgument;
+          callsGetMessage = callExpression.getMethodExpression().getCanonicalText().endsWith("getMessage");
+        }
+
+        if (message == null || "".equals(message) || callsGetMessage) {
```

Comment:
For the null / empty case, how about "Use single-argument log method instead of null/empty message"